themes/powerline-plain/powerline-plain.theme.bash

```diff
@@ -9,17 +9,7 @@ VIRTUALENV_THEME_PROMPT_COLOR=35
 
 SCM_NONE_CHAR=""
 SCM_GIT_CHAR="⎇  "
-SCM_GIT_BEHIND_CHAR="↓"
-SCM_GIT_AHEAD_CHAR="↑"
-SCM_GIT_UNTRACKED_CHAR="?:"
-SCM_GIT_UNSTAGED_CHAR="U:"
-SCM_GIT_STAGED_CHAR="S:"
-
-if [[ -z "$THEME_SCM_TAG_PREFIX" ]]; then
-    SCM_TAG_PREFIX="tag > "
-else
-    SCM_TAG_PREFIX="$THEME_SCM_TAG_PREFIX"
-fi
+SCM_GIT_SHOW_DETAILS=true
```

Comment:
The theme follows the user settings as SCM_GIT_SHOW_DETAILS is a global env var, any change done in any global env var after Bash it is loaded is overwritten, so the setting is successfully respected.